alerts/dublin_bus_transport

```diff
@@ -0,0 +1,11 @@
+---
+title: "Dublin Bus Transport integration not usable"
+created: 2023-01-26 10:46:00
+integrations:
+  - dublin_bus_transport
+github_issue: https://github.com/home-assistant/core/issues/74976
+homeassistant: ">0.30"
+---
+
+The Dublin Bus Real Time API used for this integration - https://data.dublinked.ie/cgi-bin/rtpi/realtimebusinformation?stopid=[Stop ID] - ceased working in June 2022.
+A new API is available but this integration does not use it.
```

Comment:
Alerts created issues in Home Assistant, and thus must be solvable by the end-user.

This sounds more like a bug report that needs resolving not by an end-user.

Reply:
solvable by uninstalling the broken integration - seems the maintainer/creator of the integration is not prepared to do the work to make it work with the new API.
taking this approach is in line with the Dark Skys alert, I think?

Reply:
> solvable by uninstalling the broken integration

That doesn't "solve" the issue. We also don't say: Uninstall ZHA because there is a bug.

> taking this approach is in line with the Dark Skys alert, I think?

No, Dark Sky still works, the service will be shut down in the near future (although it is unfortunately still there as an alert mainly because if has been extended a couple of times).